src/main/java/org/broadinstitute/hellbender/engine/Variant.java

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.hellbender.engine;
+
+/**
+ * Variant is (currently) a minimal variant interface needed by the Hellbender pipeline.
+ * This will be expanded as more methods are needed.
+ */
+public interface Variant {
```

Comment:
This interface should extend `Locatable`, and should also include a `getContig()` method